compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/window/ComposeView.uikit.kt

```diff
@@ -107,6 +108,8 @@ internal class ComposeView(
             try {
                 animations()
             } finally {
+                // Delay mitigates rendering glitches that can occur at the end of the animation.
+                delay(50)
```

Comment:
Adding `delay` is generally doesn't look good, so please avoid adding such questionable changes after review.

Reply:
@MatkovIvan , kk. I faced an ugly issue after rebasing. For not I don't know other ways to fix it.